Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1

```diff
@@ -195,7 +205,7 @@ function Remove-AnonymousAccessOnContainers
         break;
     }
     
-    Write-Host "Successfully validated" -ForegroundColor $([Constants]::MessageType.Update)
```

Comment:
We are just checking for User type not the RBAC role, is this pattern followed in all scripts.

Reply:
Added role check for Owner, Contributor and UAA.